packages/expo-errors/src/__tests__/Errors-test.ts

```diff
@@ -8,3 +8,71 @@ describe('UnavailabilityError', () => {
     expect(err.message).toContain('someProperty');
   });
 });
+
+describe('warnDeprecated', () => {
+  const originalWarning = console.warn;
+  function isWarnedAsync() {
+    return new Promise((resolve, reject) => {
+      console.warn = (...args: any[]) => {
+        resolve(args);
+        // return originalWarning(...args);
+      };
+    });
+  }
+  beforeEach(() => {
+    console.warn = originalWarning;
+  });
+  afterAll(() => {
```

Comment:
I think this can just be afterEach and the beforeEach call can go away entirely:
```suggestion
  afterEach(() => {
```